src/LayerPlan.h

```diff
@@ -380,14 +381,14 @@ class LayerPlan : public NoCopy
      */
     void setPrimeTowerIsPlanned(unsigned int extruder_nr);
 
-    bool getSkirtBrimIsPlanned(unsigned int extruder_nr) const
+    bool getSkirtBrimIsPlanned(const bool for_support, const unsigned int extruder_nr) const
```

Comment:
I would give `for_support` a default value `false`. Another thing is adding that extra `const` doesn't do much because those 2 variables are local copies anyway. So, I would suggest a function signature as below:
```
bool getSkirtBrimIsPlanned(unsigned int extruder_nr, bool for_support = false) const
```

Reply:
Done in https://github.com/Ultimaker/CuraEngine/pull/1024/commits/5748ac125f8e025874de29e41c4fab96a18e89ff